### PR TITLE
Installation steps of docs on Windows without using Cygwin

### LINKED ISSUE #161: Windows without Cygwin
Windows 10. (I'm assuming Linux subsystem.)

The following would probably be useful:

 - Amend the the note about Windows in the main contributor guide
    - While you are working on that, add a link to the Cygwin doc
 - Put a note at the top of the cygwin doc

### PR DESCRIPTION
**Pull request by Divya063**
Fixes #161 
Added instructions to install development environment on Windows.

### CODE REVIEW

File: cygwin.rst [hunk 1]
```diff
@@ -16,7 +16,7 @@ You need to install `Python 3 <https://www.python.org/downloads/>`_. `For instru
 
 .. _virtualenv:
 
-Virtual Environment
+Virtual Environ
```
Comment (nadaa) [inline cygwin.rst]: Why you replace Environment by Environ?

**Comment by adammichaelwood** [comment]
command line code blocks should be `none` not `rest`

**Comment by Divya063** [comment]
@adammichaelwood requested changes applied

**Comment by adammichaelwood** [comment]
I'm sorry to do this to you, but there has been a change with the terminal session code block rules.

 - bash terminal sessions should have `console` as the lang declaration
 - windows powershell should have `powershell`

Can you make the change? Thanks.

**Comment by Divya063** [comment]
Thanks for the update ,as I have mentioned command prompt in the docs (not windows powershell) so should I change command line code blocks from none to console?

**Comment by adammichaelwood** [comment]
If they use the same syntax as a bash command line, then yes use console.

**Comment by adammichaelwood** [comment]
And sorry about the change, it just happened in another issue and pull request

**Comment by Divya063** [comment]
@adammichaelwood no problem  : )

**Comment by Divya063** [comment]
@adammichaelwood Applied those changes

**Comment by adammichaelwood** [comment]
Would the commands used in Command Prompt work the same as powershell?

**Comment by adammichaelwood** [comment]
@Divya063 please see my PR on your branch

**Comment by Divya063** [comment]
@adammichaelwood No, some commands don't work the same

**Comment by adammichaelwood** [comment]
Do any of the specific commands used in the contrib guide work differently?

**Comment by Divya063** [comment]
@adammichaelwood yes command   `> odkenv\Scripts\activate`  does not work the same

screenshot 259

**Comment by Divya063** [comment]
@adammichaelwood merged the PR and thanks for the edits  : )

**Pull request merged**